Code_of_Lab5.py

```diff
@@ -0,0 +1,82 @@
+class Book:
+    def __init__(self, title='', price=0, numberOfPages=0, author='', quantity=0, numberOfSales=0):
+        self.title = title
+        self.price = price
+        self.numberOfPages = numberOfPages
+        self.author = author
+        self.quantity = quantity
+        self.numberOfSales = numberOfSales

```

Comment:
change attribute naming according to PEP 8
```suggestion
        self.number_of_sales = number_of_sales
```